integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/hooks/SimulatePushedAlarmsHooks.java

```diff
@@ -33,4 +34,47 @@ public static void simulateAlarm(
       socket.shutdownInput();
     }
   }
+
+  public static void simulateForwardedMx382Alarm(
+      final String equipmentIdentifier, final String host, final int port) throws IOException {
+
+    try (final Socket socket = new Socket(host, port)) {
+      final OutputStream outputStream = socket.getOutputStream();
+      outputStream.write(createMx382message(equipmentIdentifier));
+      socket.shutdownOutput();
+      socket.shutdownInput();
+    }
+  }
+
+  private static byte[] createMx382message(final String equipmentIdentifier) {
+    final byte[] apdu = createMx382Apdu(equipmentIdentifier);
+    final byte[] header = createMx382WpduHeader(apdu.length);
+    final byte[] wnm = new byte[header.length + apdu.length];
```

Comment:
What is `wnm`?

Reply:
Renamed it to a more logical and readable name completeMessage